sql/core/src/test/resources/sql-tests/results/typeCoercion/native/decimalArithmeticOperations.sql_spark.sql.decimalOperations.allowPrecisionLoss-false.out

```diff
@@ -0,0 +1,193 @@
+-- Automatically generated by SQLQueryTestSuite
```

Comment:
This new feature pretty looks to me. Btw, if we set multiple configurations, filenames get too long? How about adding the configuration info. in the head of files?

Reply:
Since we have two cases:
 1 - Different configs produce different results (so different files) and in this case your suggestion is fine;
 2 - Different configs produce the same results (so we have one golden file for all of them), how would you address this case?

Reply:
I was thinking the second case also output multiple **same** result files for each config.
```
// these files have the same result
subquery/in-subquery/in-joins.sql.out.1 <- sort-merge joins
subquery/in-subquery/in-joins.sql.out.2 <- hash joins
```